features/rules/GRF/GRF004_Valid-EPSG-prefix-in-coordinate-reference-system.feature

```diff
@@ -0,0 +1,35 @@
+@implementer_agreement
+@GRF
+@version1
+@E00500
+Feature: GRF004 - Valid EPSG prefix in coordinate reference system
+The rule verifies that if the name of a CRS starts with "EPSG:", it must refer to a valid code from the official EPSG geodetic parameter dataset.
+EPSG code validation is performed using the pyproj library, which includes a local copy of the official EPSG dataset (https://epsg.org) maintained by IOGP.
+For reference: https://pyproj4.github.io/pyproj/stable/api/database.html
+
+  Background: 
+
+      Given A model with Schema 'IFC4.3' or 'IFC4'
```

Comment:
```suggestion
      Given A model with Schema 'IFC4' or 'IFC4.3'
```

Sorry couldn't help myself ordering this chronologically

Reply:
Well, let's then also apply this for the[ other rules](https://github.com/buildingSMART/ifc-gherkin-rules/pull/410) and the [open GRF000 PR](https://github.com/buildingSMART/ifc-gherkin-rules/pull/409/commits/dacecccfb32f6f552cdf532aab242aacde8668d7) to be really consistent :-)